relay/kusama/src/lib.rs

```diff
@@ -1849,6 +1852,41 @@ pub mod migrations {
 		}
 	}
 
+	/// Cancel all ongoing auctions.
+	///
+	/// Any leases that come into existence after coretime was launched will not be served. Yet,
+	/// any ongoing auctions must be cancelled.
+	///
+	/// Safety:
+	///
+	/// - After coretime is launched, there are no auctions anymore. So if this forgotten to
+	/// be removed after the runtime upgrade, running this again on the next one is harmless.
+	/// - I am assuming scheduler `TaskName`s are unique, so removal of the scheduled entry
+	/// multiple times should also be fine.
+	pub struct CancelAuctions;
+	impl OnRuntimeUpgrade for CancelAuctions {
+		fn on_runtime_upgrade() -> Weight {
+			if let Err(err) = Auctions::cancel_auction(frame_system::RawOrigin::Root.into()) {
+				log::error!("Cancelling auctions failed: {:?}", err);
+			}
+			// Cancel scheduled auction as well:
+			if let Err(err) = Scheduler::cancel_named(
+				pallet_custom_origins::Origin::AuctionAdmin.into(),
```

Comment:
This does not seem to work. I am getting:

```
fellowship-runtimes/relay/kusama/src/lib.rs:1871:96:
    called `Result::unwrap()` on an `Err` value: Module(ModuleError { index: 29, error: [1, 0, 0, 0], message: Some("NotFound") })
```
Does not work with `cancel` directly either. :thinking:

Reply:
It gets executed multiple times with try-runtime - that's where this error is coming from.